algolia/opt/composable_filter.go

```diff
@@ -84,7 +84,27 @@ func (o *composableFilterOption) UnmarshalJSON(data []byte) error {
 
 	if json.Unmarshal(data, &filter) == nil {
 		ok = true
-		ors = strings.Split(filter, ",")
+		replacer := strings.NewReplacer("(", " ", ")", " ")
```

Comment:
What about the `[`, I see that it's supported in the tests, are they handled elsewhere?

Reply:
Yes, they are handled as slices because `Unmarshal` will not unmarshal that as a string and they will be handled in the below.

Reply:
Ahh of course :)